ev3dev2/motor.py

```diff
@@ -1638,11 +1699,26 @@ def reset(self, motors=None):
             motor.reset()
 
     def stop(self, motors=None):
+        """
+        stop the motors without setting the stop_action (brake, hold, coast)
```

Comment:
Could you explain what this actually means to someone who doesn't know how `stop_action`s work?

Reply:
How about:

Stop the motors without specifying the “stop action”.  “stop actions” are one of the following 
- coast to a stop
- apply the brake to stop but do not hold the motor in position once stopped
- apply the brake to stop and hold the motor in position once stopped

Reply:
The behavior here is that it will use the last-used `stop_action`, right? I think that's the key here. And honestly, I'm slightly worried about the fact that this is public API surface area if its purpose isn't necessarily clear.

Reply:
Right...which could lead to some unexpected behavior. What do you say we
- delete off()
- stop() needs to accept an arg to specify the stop_action. I think we just have to document the heck out of the three options.

Reply:
Filed #540 for this